web/models/character.ex

```diff
@@ -0,0 +1,29 @@
+defmodule Yggdrasil.Character do
+  use Yggdrasil.Web, :model
+
+  schema "characters" do
+    field :name, :string
+    belongs_to :user, Yggdrasil.User
+    belongs_to :game, Yggdrasil.Game
+
+    timestamps
+  end
+
+  @required_fields ~w(name user_id game_id)
```

Comment:
Very good point, the only tests that hint to this are the ones around the assoc constraints and it just sort eludes to it so definitely need those cases in there prove they are required.

As far as why *_id fields, because the cast works with ids not the structs for now. I believe with ecto 2.0 - [this](https://github.com/elixir-lang/ecto/blob/master/CHANGELOG.md#improved-association-support) will enable what us to use the game/user fields instead of having to use ids.

from my testing for example this does not work assuming the required fields was adjusted to mark game/user instead of the ids like this `~w(name user game)`
```elixir
test "valid character produces a valid changeset" do
    fixture = do_setup

    user = fixture.user
    game = fixture.game

    valid_character = %{@character | user: user, game: game}

    character = Character.changeset(%Character{}, valid_character)
    assert character.valid? == true
  end
```
That will fail because the cast will not find the association or something like that. There are also put_assoc functions etc, but they do not work with belongs_to for now anyway tried those as well. I think 2.0 will iron this out once it lands.